advisor/2016-07-12-preview/swagger/advisor.json

```diff
@@ -0,0 +1,461 @@
+{
+	"swagger": "2.0",
+	"info": {
+		"version": "v1",
+		"title": "Microsoft.Azure.Engineering.Gateway"
+	},
+	"host": "management.azure.com",
+	"schemes": ["https"],
+	"paths": {
+		"/subscriptions/{subscriptionId}/providers/microsoft.Advisor/generaterecommendations": {
+			"post": {
+				"tags": ["GenerateRecommendations"],
+				"operationId": "GenerateRecommendations_POST",
+				"consumes": [],
+				"produces": ["application/json",
+				"text/json"],
+				"parameters": [{
+					"name": "subscriptionId",
+					"in": "path",
+					"required": true,
+					"type": "string",
+					"format": "uuid"
+				},
+				{
+					"name": "api-version",
+					"in": "query",
+					"required": true,
+					"type": "string"
+				}],
+				"responses": {
+					"200": {
+						"description": "OK",
+						"schema": {
+							"type": "object"
```

Comment:
having an model definition to serialize into would be useful rather than having it in `object`